webidl/manifest.webidl

```diff
@@ -1,15 +1,46 @@
 dictionary WebPublicationManifest {
-    required DOMString                         url;
-             DOMString                         lang;
-             TextDirection                     direction = "auto";
-             TextDirection                     readingProgression = "auto";
-             sequence<LocalizableString>       name;
-             DOMString                         id;
-             sequence<Contributor>             authors;
-             DOMString                         dateModified;
-             DOMString                         datePublished;
-             sequence<PublicationLink>         links;
-             sequence<PublicationLink>         readingOrder;
-             sequence<PublicationLink>         resources;
-             sequence<PublicationLink>         toc;
+    required DOMString                          url;
+    required DOMString                          type;
+
+             DOMString                          accessMode;
+             DOMString                          accessModeSufficient;
+             DOMString                          accessibilityAPI;
+             DOMString                          accessibilityControl;
+             DOMString                          accessibilityFeature;
```

Comment:
This is expressed using an array of strings on the example available at schema.org: https://schema.org/accessibilityFeature

Reply:
Changed in https://github.com/w3c/wpub/pull/285/commits/aefe40f27a55ebf37d4272c8d887d32d5bcd2bf3